x/emissions/module/rewards/rewards.go

```diff
@@ -66,86 +66,48 @@ func EmitRewards(
 		return errors.Wrapf(err, "failed to calculate topic rewards")
 	}
 
-	// Calculate then pay out topic rewards to topic participants
-	totalRewardToStakedReputers := alloraMath.ZeroDec() // This is used to communicate with the mint module
-	for _, topicId := range sortedRewardableTopics {
-		Logger(ctx).Debug(fmt.Sprintf("Distributing rewards for topic %d", topicId))
```

Comment:
why remove logs like this?

Reply:
it's been moved inside the `getDistributionAndPayoutRewardsToTopicActors` function